mailbox/cassandra/src/main/java/org/apache/james/mailbox/cassandra/modules/CassandraAclModule.java

```diff
@@ -33,14 +33,15 @@
 import org.apache.james.mailbox.cassandra.table.CassandraACLTable;
 
 import com.datastax.driver.core.schemabuilder.SchemaBuilder;
+import com.google.common.collect.ImmutableList;
 
 public class CassandraAclModule implements CassandraModule {
 
     private final List<CassandraTable> tables;
     private final List<CassandraType> types;
 
     public CassandraAclModule() {
-        tables = Collections.singletonList(
```

Comment:
And so what? I prefer saying we use only one list implementation for this, that embed the fact it's immutable in its name.